google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -263,6 +255,17 @@ public void run() {
     return outstandingPublish.publishResult;
   }
 
+  /**
+   * There may be non-recoverable problems with a request for an ordering key. In that case, all
+   * subsequent requests will fail until this method is called.
+   *
```

Comment:
Can you add a comment that calling this function when the key has not been failed is no-op?

Reply:
Done.